docs/src/pages/docs/functions/logic-functions.md

```diff
@@ -263,6 +263,68 @@ back a predicate function ready for insertion into your flow. All predicate
 based functions in `crocks` take either a [`Pred`][pred] or predicate
 function, so it should be easy to swap between the two.
 
+```javascript
+import not from 'crocks/logic/not'
+
+import Pred from 'crocks/Pred'
+import propSatisfies from 'crocks/predicates/propSatisfies'
+import isString from 'crocks/predicates/isString'
+import and from 'crocks/logic/and'
+import compose from 'crocks/core/compose'
+import flip from 'crocks/combinators/flip'
+import identity from 'crocks/combinators/identity'
+
+// isFalsy :: a -> Boolean
+const isFalsy = not(identity)
+
+// isTruthy :: a -> Boolean
+const isTruthy = not(isFalsy)
+
+isTruthy('Test string')
+//=> true
+
+isTruthy('')
+//=> false
+
+isFalsy('')
+//=> true
+
+isFalsy('Test string')
+//=> false
+
+// User :: { String, String, String}
```

Comment:
A proper record def should define the expected shape like:
```javascript
// User :: { email: String, firstName: String, lastName: String }
```

Reply:
Thanks, i was curious how to do this :smile: